aci_interface_policies.tf

```diff
@@ -181,17 +181,18 @@ locals {
   new_leaf_interface_configuration = flatten([
     for node in local.nodes : [
       for interface in try(node.interfaces, []) : {
-        key               = format("%s/%s/%s", node.id, try(interface.module, local.defaults.apic.interface_policies.nodes.interfaces.module), interface.port)
-        node_id           = node.id
-        module            = try(interface.module, local.defaults.apic.interface_policies.nodes.interfaces.module)
-        port              = interface.port
-        policy_group_type = try([for pg in local.access_policies.leaf_interface_policy_groups : pg.type if pg.name == interface.policy_group][0], "access")
-        policy_group      = try("${interface.policy_group}${local.defaults.apic.access_policies.leaf_interface_policy_groups.name_suffix}", "system-ports-default")
-        breakout          = try(interface.breakout, "none")
-        fex_id            = try(interface.fex_id, "unspecified")
-        description       = try(interface.description, "")
-        shutdown          = try(interface.shutdown, false)
-        role              = node.role
+        key                        = format("%s/%s/%s", node.id, try(interface.module, local.defaults.apic.interface_policies.nodes.interfaces.module), interface.port)
+        node_id                    = node.id
+        module                     = try(interface.module, local.defaults.apic.interface_policies.nodes.interfaces.module)
+        port                       = interface.port
+        policy_group_type          = try([for pg in local.access_policies.leaf_interface_policy_groups : pg.type if pg.name == interface.policy_group][0], "access")
+        policy_group               = try("${interface.policy_group}${local.defaults.apic.access_policies.leaf_interface_policy_groups.name_suffix}", "system-ports-default")
+        breakout                   = try(interface.breakout, "none")
+        fex_id                     = try(interface.fex_id, "unspecified")
+        description                = try(interface.description, "")
+        shutdown                   = try(interface.shutdown, false)
+        role                       = node.role
+        port_channel_member_policy = try(interface.port_channel_member_policy, "")
```

Comment:
Add name_suffix logic here.

Reply:
moved to locals for all 3 occurences (interface, subinterface, fexinterface)